demisto_sdk/commands/init/initiator.py

```diff
@@ -240,7 +243,7 @@ def pack_init(self) -> bool:
         if create_integration in ['y', 'yes']:
             integration_init = Initiator(output=os.path.join(self.full_output_path, 'Integrations'),
                                          integration=True, common_server=self.common_server,
-                                         demisto_mock=self.demisto_mock, template=self.template)
+                                         demisto_mock=self.demisto_mock, template=self.template, category=self.category)
```

Comment:
might be smarter to add a question to the user (using `input`) of which integration category does he want to add

Reply:
done.